From 448527f765b1c4f9dc5452cc069baddecaf55e7e Mon Sep 17 00:00:00 2001
From: Antonio Rojas <arojas@archlinux.org>
Date: Wed, 21 Feb 2018 13:57:28 +0100
Subject: [PATCH 1/8] Install apper binary as TARGET instead of PROGRAM

Otherwise the RPATH set in add_target_properties is ignored

BUG: 390836
Differential Revision: https://phabricator.kde.org/D10706
---
 Apper/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Apper/CMakeLists.txt b/Apper/CMakeLists.txt
index 2b71603..a9855a9 100644
--- a/Apper/CMakeLists.txt
+++ b/Apper/CMakeLists.txt
@@ -49,7 +49,7 @@ target_link_libraries(apper
 
 set_target_properties(apper PROPERTIES INSTALL_RPATH ${CMAKE_INSTALL_FULL_LIBDIR}/apper)
 
-install(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/apper DESTINATION ${CMAKE_INSTALL_BINDIR})
+install(TARGETS apper DESTINATION ${CMAKE_INSTALL_BINDIR})
 install(FILES org.kde.apper.desktop DESTINATION ${CMAKE_INSTALL_APPDIR})
 install(FILES org.kde.apper_installer.desktop DESTINATION ${CMAKE_INSTALL_APPDIR})
 install(FILES org.kde.apper_settings.desktop DESTINATION ${CMAKE_INSTALL_APPDIR})
-- 
2.17.1

